### PR TITLE
Add repair type specific SOR configurations

### PR DESCRIPTION
**Pull request by Bhavesh-Patel**

### CODE REVIEW

File: delayGitPush.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sleep 900
+git push -f
+
```
Comment (Renny4Real) [inline delayGitPush.sh:3]: Ooooh. What's this do? 😀

**Comment by Bhavesh-Patel** [comment]
[Use Newark's options for communal journey](https://trello.com/c/V1HhxUBh/223-use-newarks-options-for-communal-journey)

File: delayGitPush.sh (hunk 1 above)
Comment (Renny4Real) [inline delayGitPush.sh:3]: Ooooh. What's this do? 😀

Comment (Bhavesh-Patel) [reply delayGitPush.sh]: Oops...Removed 😁

**Review by Renny4Real — APPROVED** [review]

**Pull request merged**